Classes/Command/InstallCommandController.php

```diff
@@ -41,24 +42,24 @@ class InstallCommandController extends CommandController
     protected $packageStatesGenerator;
 
     /**
-     * TYPO3 Setup
+     * TYPO3 Setup.
```

Comment:
Full stops in the headline have been removed in #256 and should stay removed.

The docheader comments are used for documentation **and** to show the help texts. Having them consistent, matters